config/locales/ja.yml

```diff
@@ -284,8 +310,11 @@ ja:
     show_more: もっと見る
     visibilities:
       private: 非公開 - フォロワーだけに公開
```

Comment:
I suggest removing `- フォロワーだけに公開` because it could be verbose when displayed with `private_long`. (and I think it is expected to be - see #2111.).

`public`, `unlisted` should also be updated.